cmake/WASINNDeps.cmake

```diff
@@ -357,6 +388,12 @@ function(wasmedge_setup_llama_target target)
     if(WASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_CUBLAS)
       set_property(TARGET ggml-cuda PROPERTY POSITION_INDEPENDENT_CODE ON)
     endif()
+    if(WASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_HIP)
+      # Some upstream revisions expose ggml-hip; ignore if not present.
```

Comment:
I don't get the point of this comment.  
We should expect the upstream to expose `ggml-hip`. Could you confirm whether the latest llama.cpp includes this target?